core/model/devicetokens.py

```diff
@@ -26,7 +26,7 @@ class DeviceToken(Base):
     id = Column("id", Integer, primary_key=True)
     patron_id = Column(
         Integer,
-        ForeignKey("patrons.id", ondelete="CASCADE"),
+        ForeignKey("patrons.id", ondelete="CASCADE", name="devicetokens_patron_fkey"),
```

Comment:
This change seems unrelated, why the change here?

Reply:
ahh I see in the PR description: 
> Added the fkey name to the devicetokens model to avoid unnecessary migration

I'm not sure that is true, or at least true anymore. You have to be somewhat careful with Alembic because it generates migrations based on the current state of your database. Is it possible your database was somewhat out of sync when you needed to add this foreigin key name?

Reply:
I stand corrected, I did some more testing, and you are right @RishiDiwanTT this is required. Will resolve this comment.